test/on.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var S = require('..');
+
+var eq = require('./internal/eq');
+var rem = require('./internal/rem');
+
+
+describe('on', function() {
+
+  it('is a quaternary function', function() {
+    eq(typeof S.on, 'function');
+    eq(S.on.length, 4);
+  });
+
+  it('should pass the last two arguments through the modifying function into the converging function', function() {
```

Comment:
Nit: I prefer _it does X_ to _it should do X_. In this case, _it passes the last two arguments…_